packages/overlay/test/animations.test.js

```diff
@@ -99,35 +99,27 @@ customElements.define(
 
 function afterOverlayOpeningFinished(overlay, callback) {
   const observer = new MutationObserver((mutations, observer) => {
-    for (let i = 0; i < mutations.length; i++) {
-      const mutation = mutations[i];
-      if (mutation.attributeName === 'opening') {
-        const target = mutation.target;
-        const hasFinishedOpening = target.hasAttribute('opened') && !target.hasAttribute('opening');
-        if (hasFinishedOpening) {
-          observer.disconnect();
-          afterNextRender(overlay, callback);
-          return;
-        }
-      }
+    const isOverlayOpened = mutations.some(({ target }) => {
+      return target.hasAttribute('opened') && !target.hasAttribute('opening');
```

Comment:
**note:** There is no need to check the attribute name since the observer has been already created with an attribute filter.